zipline/data/data_portal.py

```diff
@@ -59,6 +59,362 @@
 HISTORY_FREQUENCIES = set(["1m", "1d"])
 
 
+class DailyHistoryAggregator(object):
```

Comment:
stylistic thing, can you move this into its own file?  when I open `data_portal.py`, I prefer immediately seeing the `DataPortal` code.